provider/pkg/metadata/metadata.go

```diff
@@ -22,6 +22,7 @@ type CloudAPIResource struct {
 	AutoNamingSpec    *AutoNamingSpec                 `json:"autoNamingSpec,omitempty"`
 	Required          []string                        `json:"required,omitempty"`
 	CreateOnly        []string                        `json:"createOnly,omitempty"`
+	ReadOnly          []string                        `json:"readOnly,omitempty"`
```

Comment:
Here is an opportunity for a documentation comment on what this is about and a few examples of how this data looks like, specifically that nested names will be separated by "/", and that the names themselves are using Pulumi notation e.g. groupArn. If array element separator is used (is it '*')? also good to document there. It is still a little weird that "/" is the separator but that we can live with. 

Having a doc comment will save future maintainers some jumping around! Thank you.

Reply:
Added some docs.